### ISSUE TITLE
Allow for limited markdown styling in #flashcard headings

### ISSUE THREAD

**Issue by orionrush** [NeuraCache/flashcards#2]
Allow for markdown _italics_, **bold** and `code` in #flashcard headings.

```
_italic_, *italic*
__bold__, **bold**
`code`
```

```
When does `this` refer to the *Global Object*? #flashcard
// question text
---
```

Though, _italic_ and `code` may be sufficient for 90% of use cases. As the heading itself is already rendered bold in NueraCache creating yet bolder text may cause other UI legibility issues.

**Comment by NeuraCache** [comment]
I'm working on a rich question feature which will enable any content as a question (images, latex, italic bold, etc)

**Comment by floofy-moo** [comment]
@NeuraCache  Hey did ^ get enabled for questions? I have latex as the question and only the answer gets rendered as such.

**Comment by NeuraCache** [comment]
this is now available on iOS 2.19 along with a new two-sided card spec 👍
please see the updated wiki 
https://wiki.neuracache.com/markdown-flashcards
